react/ShelfItem.js

```diff
@@ -10,9 +11,19 @@ import { changeImageUrlSize, toHttps } from './utils/urlHelpers'
  * ShelfItem Component. Normalizes the item received in the props
  * to adapt to the extension point prop.
  */
-export default class ShelfItem extends Component {
+class ShelfItem extends Component {
   static propTypes = shelfItemPropTypes
 
+  sendProductClickEvent = (product) => {
+    const _sendProductClickEvent = () => {
```

Comment:
why do you need this?

Reply:
Because ProductSummary expects a function as the type of _actionOnClick_. I use a inner function to pass the product param in the function declartion.